ocaml/xapi/xapi_pool_update.ml

```diff
@@ -302,7 +302,25 @@ let verify update_info update_path =
     ) [update_xml_path; repomd_xml_path];
   debug "Verify signature OK for pool update uuid: %s by key: %s" update_info.uuid update_info.key
 
+let patch_uuid_of_update_uuid uuid =
+  let arr = Uuid.int_array_of_uuid (Uuid.uuid_of_string uuid) in
```

Comment:
Right, I hadn't forgotten that.

Fair enough, let's go for it like this (after squashing).